common/src/main/scala/react/floatingui/Tooltip.scala

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package react.floatingui
+
+import cats.syntax.all.*
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.feature.ReactFragment
+import japgolly.scalajs.react.vdom.TagOf
+import japgolly.scalajs.react.vdom.html_<^._
+import lucuma.ui.syntax.all.*
+import lucuma.ui.syntax.all.given
+import org.scalajs.dom
+import org.scalajs.dom.html
+import react.common.ReactFnProps
+import react.common.Style
+import react.floatingui.hooks._
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSExportTopLevel
+import scala.scalajs.js.|
+
+import js.annotation._
+
+/**
+ * Tooltip base on floating ui see: https://floating-ui.com/docs/react-dom
+ */
+case class Tooltip(trigger: VdomTag, tooltip: VdomNode, placement: Placement = Placement.Top)
```

Comment:
I wonder if we should have a `disabled` flag or something? I'm thinking of a situation where we have a element that will expect to always be in the DOM, but will only have a tooltip some of the time.

Another situation could be where we want the tooltip to always be displayed. For example, the way we currently display errors for inputs.

Reply:
Right, we could add those as needed